kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/base/ThrowableUtil.java

```diff
@@ -31,9 +32,11 @@ private ThrowableUtil() {
      *
      * @param throwable
      *            the throwable to propagate
+     * @throws RuntimeException
+     *             translates the provided exception as a runtime exception
      */
     public static void propagate(final Throwable throwable) {
-        throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, throwable);
+        throw new RuntimeException(throwable);
```

Comment:
This has been written for the same reason to translate a checked exception to an unchecked one. If the caller doesn't have any idea about how to recover, it is beneficial to propagate it as an unchecked one.